pyproject.toml

```diff
@@ -4,21 +4,19 @@ version = "0.1.0"
 readme = "README.md"
 requires-python = ">=3.12"
 dependencies = [
+    "bidict>=0.23.1",
     "coqpyt",
     "mysql-connector-python>=9.1.0",
     "python-dotenv>=1.0.1",
 ]
 
 [project.scripts]
-main = "atomization.atomizer:main"
-dry = "atomization.atomizer_dry:main"
-test = "pytest:main"  
+atomize = "atomization.atomizer:main"
```

Comment:
This requires that you edit [entrypoint](https://github.com/Beneficial-AI-Foundation/atomization/blob/main/docker/entrypoint.sh)

Reply:
with what though? show me tomorrow?

Reply:
main -> atomize in entrypoint.sh